app/controllers/api/v3/courses_controller.rb

```diff
@@ -24,11 +36,19 @@ def show
 
     private
 
+      # Used for sitemap on Find
+      # When these fields are specified return all results for sitemap
```

Comment:
Could this comment be removed by improving the readability of the code?

Reply:
done